src/Stripe.net/Infrastructure/Public/Mapper.cs

```diff
@@ -73,5 +67,17 @@ private static void ApplyStripeResponse(string json, StripeResponse stripeRespon
 
             stripeResponse.ObjectJson = json;
         }
+
+        private static JsonSerializerSettings InitSerializerSettings()
+        {
+            return new JsonSerializerSettings
+            {
+                Converters = new List<JsonConverter>
+                {
+                    new StripeObjectConverter(),
+                },
+                DateParseHandling = DateParseHandling.None,
```

Comment:
Didn't you need to set something related to `CultureInfo` here too?

Reply:
I'm not sure how Newtonsoft.Json uses the culture exactly (except for formatting date strings, but that's irrelevant since we're specifically disabling that feature) but according to the [docs](https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_JsonSerializerSettings_Culture.htm), the default value is `InvariantCulture` which is probably what we want.